### DISCUSSION TITLE
Using wireless adapter as second NIC

### DISCUSSION THREAD

**Discussion by DevEsteves** [Security-Onion-Solutions/securityonion#11381] [Unsupported Versions] [answered]
I need two NICs to run Security Onion, can I use a wireless adapter as the second NIC, or does it have to be an Ethernet NIC?

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.4/hardware.html#nic:
> Security Onion is designed to use wired interfaces. You may be able to make wireless interfaces work, but we don’t recommend or support it.
